app/controllers/items_controller.rb

```diff
@@ -243,6 +233,13 @@ def authorize_manage!
     authorize! :update, @cocina
   end
 
+  def build_argo_show_presenter
+    ArgoShowPresenter.new(nil, nil, nil).tap do |presenter|
```

Comment:
This is just enough of an ArgoShowPresenter to render the components.

Reply:
Hmmm, is the change set no longer needed within e.g. `Show::Item::AccessRightsComponent`? That component looks like it needs the change set to for `:view_access, :download_access, :access_location, :controlled_digital_lending`

Reply:
Change set is constructed from the cocina here https://github.com/sul-dlss/argo/blob/main/app/presenters/argo_show_presenter.rb#L12-L14